optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/impl/statistic/StatisticUtilsTest.java

```diff
@@ -0,0 +1,87 @@
+import org.optaplanner.core.api.score.buildin.simplelong.SimpleLongScore;
+
+import static org.junit.Assert.*;
+
+import static org.mockito.Mockito.*;
+
+public class StatisticUtilsTest {
+
+    private static final double DELTA = 0.001d;
+
+    @Test
+    public void testSingleDetermineStandardDeviationDoubles() throws Exception {
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = Arrays.asList(createSubSingleBenchmarkResult(SimpleScore.valueOf(0), 0));
+        assertArrayEquals(new double[]{0d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, SimpleScore.valueOf(0), subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testMultipleDetermineStandardDeviationDoubles() throws Exception {
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = new ArrayList<SubSingleBenchmarkResult>(2);
+        subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(SimpleScore.valueOf(-2), 0));
+        subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(SimpleScore.valueOf(-4), 1));
+        assertArrayEquals(new double[]{1d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, SimpleScore.valueOf(-3), subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testLargeDetermineStandardDeviationDoubles() throws Exception {
+        long[] subSingleBenchmarkScores = new long[]{-19289560268L, -19345935795L, -19715516752L, -19589259253L, -19390707618L, -19641410518L};
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = new ArrayList<SubSingleBenchmarkResult>(6);
+        SimpleLongScore averageScore = SimpleLongScore.valueOf(0);
+        for (int i = 0; i < subSingleBenchmarkScores.length; i++) {
+            SimpleLongScore current = SimpleLongScore.valueOf(subSingleBenchmarkScores[i]);
+            subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(current, i));
+            averageScore = averageScore.add(current);
+        }
+        averageScore = averageScore.divide(subSingleBenchmarkScores.length);
+        assertArrayEquals(new double[]{160338212.294d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, averageScore, subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testGetStandardDeviationString() throws Exception {
+        assertEquals(null, StatisticUtils.getStandardDeviationString(null));
+        assertEquals("2.0", StatisticUtils.getStandardDeviationString(new double[]{2d}));
+        assertEquals("1.41", StatisticUtils.getStandardDeviationString(new double[]{Math.sqrt(2)}));
+        assertEquals("1.60E8", StatisticUtils.getStandardDeviationString(new double[]{160338212.294d}));
+        assertEquals("2.0E9", StatisticUtils.getStandardDeviationString(new double[]{2000000000d}));
+        assertEquals("2.0E10", StatisticUtils.getStandardDeviationString(new double[]{20000000000d}));
```

Comment:
Because the comment doesn't motivate why it should be locale insensitive, git annotate doesn't refresh my memory and I currently believe the opposite is true, I think we should remove it now and print a locale sensitive value. Wdyt?